Cargo.toml

```diff
@@ -9,12 +9,18 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 anyhow = "1.0.75"
-bevy = { version = "0.11.2", default-features = false }
+bevy = { version = "0.11.2", default-features = false, features = ["bevy_audio"] }
```

Comment:
We should add a test crate that depends on this crate to run unit tests and ensure the crate functions correctly as a library. With just examples, I assume this mistake wouldn't have been obvious.

Reply:
I haven't done much unit testing before (let alone with Rust) but that sounds reasonable